ci/config.yml

```diff
@@ -0,0 +1,85 @@
+# Java Gradle CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-java/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: ojhughes/spring-cloud-app-broker-ci:v1
```

Comment:
Can we use the account we have under `springcloudservices`?

Reply:
Sure